currency_rate_update_oxr/models/res_company.py

```diff
@@ -10,3 +10,7 @@ class ResCompany(models.Model):
     openexchangerates_app_id = fields.Char(
         string='OpenExchangeRates.org App ID',
     )
+    eod_rates = fields.Boolean(
```

Comment:
Since this is a oxr related field, should be named `openexchangerates_eod_rates`

Reply:
Thanks! Updated.